src/pages/authPages/Login.tsx

```diff
@@ -1,8 +1,42 @@
 import React, { FC } from 'react';
 import { Box, TextField, Button, Typography, Container, Paper } from '@mui/material';
 import NavigationBar from '../homePage/NavigationBar';
+import * as yup from "yup"
+import { useAuthContext } from '../../context/AuthContext';
+import { yupResolver } from '@hookform/resolvers/yup'
+import { useForm } from 'react-hook-form'
+import { useNavigate } from 'react-router-dom';
+
+interface LoginData {
+    email?: string,
+    password: string,
+}
+const schema = yup.object({
+    email: yup.string().email('Email is required'),
+    password: yup.string().required('Password is required').min(8, 'Password is too short - should be 8 chars minimum.')
+})
 
 const LoginPage: FC = () => {
+    const { loginClient } = useAuthContext()
+    const { register, handleSubmit, formState: { errors } } = useForm<LoginData>({ resolver: yupResolver(schema) })
+    const navigate = useNavigate();
+
+
+    const onSubmit = async (values: LoginData) => {
+        const { email, password } = values
+        const userLogin: LoginData = { email, password }
+
+        await loginClient(userLogin)
+            .then(() => {
+                console.log('Successfully logged in')
+                navigate('/')
+            }).catch(() => {
+                console.log('Wrong login or password provided')
```

Comment:
obsluga bledow to nie console.log :D ale rozumiem, ze to chwilowe poki nie masz AlertProvider?